woocommerce/woocommerce-android
# SKU search in product selector

Closes: #9150


### Description

This PR adds the possibility to search products by SKU (partial match) in the product selector. The UX follows the iOS implementation as described here: pdfdoF-2Wl-p2

### Testing instructions
1. Create a new order and click the "Add products" button to open the product selector.
2. Verify that when the search field is active the search type options appear.
3. Verify that it's possible to search products by SKU.

### Images/gif

<img src=https://github.com/woocommerce/woocommerce-android/assets/4527432/a7473853-0661-46a0-a5dd-f377557493b1 height=600/>
